packages/docs/docs/zh/plugin/official/plugin-nprogress.md

```diff
@@ -5,7 +5,7 @@ metaTitle: Nprogress 插件 | VuePress
 
 # [@vuepress/plugin-nprogress](https://github.com/vuejs/vuepress/tree/master/packages/%40vuepress/plugin-nprogress)
 
-> Nprogress plugin
+一个基于 [nprogress](https://github.com/rstacruz/nprogress) 的进度条插件。<Badge text="1.0.0-alpha.31+" vertical="baseline"/>
```

Comment:
I don't think that we should use `vertical-align: baseline ` here, and the default `top` is intentional.